docs/reference/config-params.md

```diff
@@ -842,6 +842,21 @@ services:
       ENV_VAR_A: some-literal-value  # Literal value
 ```
 
+When there is a need to reference any dependent environment variables it can be achieved by using double curly braces
+
+> Environment variable with as literal string referencing dependent environment variables:
+```yaml
+version: 3.7
+services:
+  my-service:
+    labels:
+      ...
+    environment:
+      ENV_VAR_A: foo
+      ENV_VAR_B: bar
+      ENV_VAR_C: {{ENV_VAR_A}}/{{ENV_VAR_B}}  # referencing other dependent environment variables
+```
+
```

Comment:
This make a lot of sense.

In fact, we need this as a custom method for relying on ENV values, that will pick the value of another ENV value. Esp., that compose doesn't have a native mechanism to do this.